packages/react/src/components/Attachments/AttachmentWindow.js

```diff
@@ -15,6 +15,8 @@ function AttachmentWindow() {
   const toggle = useAttachmentWindowStore((state) => state.toggle);
   const data = useAttachmentWindowStore((state) => state.data);
   const setData = useAttachmentWindowStore((state) => state.setData);
+  const isPending = useAttachmentWindowStore((state) => state.isPending);
```

Comment:
Do we need global state here ? Why not use `useState` for this ?

Reply:
done!